.circleci/verbatim-sources/workflows-ecr-gc.yml

```diff
@@ -10,7 +10,7 @@
       - ecr_gc_job:
             name: ecr_gc_job_for_pytorch
             project: pytorch
-            tags_to_keep: "271,262,256,278,282,291,300,323,327,347,389,401,402,403,405,a8006f9a-272d-4478-b137-d121c6f05c83,07597f23-fa81-474c-8bef-5c8a91b50595"
+            tags_to_keep: "271,262,256,278,282,291,300,323,327,347,389,401,402,403,405,a8006f9a-272d-4478-b137-d121c6f05c83,6e7b11da-a919-49e5-b2ba-da66e3d4bb0a"
```

Comment:
Are you sure you want to delete the old tag?

Reply:
Yup, they should all be updated to the new tag.

Reply:
@seemethere Hmm, let me clarify some more. We like to keep old tags around for a little beyond when we retire them, in case there's an emergency and we need to roll them back. Given that knowledge, are you sure you want to delete the old tag? :)

Reply:
Will add the old tag back